monome/teletype
# fixed R... + reversed N.B and N.BX

#### What does this PR do?
Fix `R...` and change the bit-order of OPs N.B and N.BX, by request the number of scales is bumped from 8 to 16

#### Provide links to any related discussion on [lines](https://llllllll.co/).
https://llllllll.co/t/teletype-3-2-feature-requests-and-discussions/32052/343

#### How should this be manually tested?
`B100 = R001`
`B110 = R011`

`N.B 0 R101011010101` should setup C-major-scale
`N.B 0 -1` should setup C-dorian

#### Any background context you want to provide?
N/A

#### If the related Github issues aren't referenced in your commits, please link to them here.
N/A

#### I have,
